docs/REST API/rest-api.md

```diff
@@ -1266,6 +1270,10 @@ Modifies the Fleet's configuration with the supplied information.
       }
     ],
     "ndes_scep_proxy": null,
+    "change_management": {
```

Comment:
@rachaelshaw this strikes me as a top-level setting, on par with `webhook_settings`, `logging`, etc. Are you sure we want this field to be under `integrations`?

Reply:
To document, Rachael agrees and will update this draft PR